src/js/vre/chapter31/config/form.js

```diff
@@ -63,17 +64,7 @@ const formConfig = {
     veteranInformation: {
       title: 'Veteran Information',
       pages: {
-        veteranInformation: createVeteranInfoPage(fullSchema31, {
-          uiSchema: {
-            vaRecordsOffice: {
-              'ui:title': 'VA benefit office where your records are located',
-              'ui:help': facilityLocatorLink
-            }
-          },
-          schema: {
-            vaRecordsOffice
-          }
-        })
+        veteranInformation: createVeteranInfoPage(fullSchema31)
```

Comment:
What's the threshold for creating something like this? e.g. if the identical page is used on more than one form?

Reply:
We don't have a hard rule about it, but generally if the page has more than a couple fields, has some specific validation/display logic, and is used on more than one form it's a good candidate for being pulled out like this.

We've been more aggressive about it in the past, but it can lead to complicated page definitions that try to account for variation between forms.